.gitignore

```diff
@@ -18,3 +18,4 @@ confxml.h
 *.obj
 *.so
 *.rcc
+.vscode/settings.json
```

Comment:
Unrelated to the PR and IDE ignores should be put into your local `gitignore` (e.g. `~/.config/git/ignore`).

Reply:
This should be reverted now.